modin/test/test_partition_api.py

```diff
@@ -54,27 +54,50 @@
 
 
 @pytest.mark.parametrize("axis", [None, 0, 1])
-def test_unwrap_partitions(axis):
+@pytest.mark.parametrize("reset_index", [True, False])
+@pytest.mark.parametrize("reset_columns", [True, False])
+def test_unwrap_partitions(axis, reset_index, reset_columns):
```

Comment:
maybe we should name those `reverse_index` and `reverse_columns` to better reflect what they're doing?